jira/client.py

```diff
@@ -1203,7 +1213,240 @@ def dashboard(self, id: str) -> Dashboard:
+        dashboard.gadgets.extend(self.dashboard_gadgets(id) or [])
+        return dashboard
+
+    @cloud
+    @experimental
+    def create_dashboard(
+        self,
+        name: str,
+        description: str | None = None,
+        edit_permissions: list[dict] | list = [],
+        share_permissions: list[dict] | list = [],
+    ) -> Dashboard:
+        """Create a new dashboard and return a dashboard resource for it.
+
+        Args:
+            name (str): Name of the new dashboard `required`.
+            description (Optional[str]): Useful human-readable description of the new dashboard.
+            edit_permissions (list | list[dict]): A list of permissions dicts `required`
+                though can be an empty list.
+            share_permissions (list | list[dict]): A list of permissions dicts `required`
+                though can be an empty list.
+
+        Returns:
+            Dashboard
+        """
+        data: dict[str, Any] = remove_empty_attributes(
+            {
+                "name": name,
+                "editPermissions": edit_permissions,
+                "sharePermissions": share_permissions,
+                "description": description,
+            }
+        )
+        url = self._get_url("dashboard")
+        r = self._session.post(url, data=json.dumps(data))
+
+        raw_dashboard_json: dict[str, Any] = json_loads(r)
+        return Dashboard(self._options, self._session, raw=raw_dashboard_json)
+
+    @cloud
+    @experimental
+    def copy_dashboard(
+        self,
+        id: str,
+        name: str,
+        description: str | None = None,
+        edit_permissions: list[dict] | list = [],
+        share_permissions: list[dict] | list = [],
+    ) -> Dashboard:
+        """Copy an existing dashboard.
+
+        Args:
+          id (str): The ``id`` of the ``Dashboard`` to copy.
+          name (str): Name of the new dashboard `required`.
+          description (Optional[str]): Useful human-readable description of the new dashboard.
+          edit_permissions (list | list[dict]): A list of permissions dicts `required`
+            though can be an empty list.
+          share_permissions (list | list[dict]): A list of permissions dicts `required`
+            though can be an empty list.
+
+
+        """
+        data: dict[str, Any] = remove_empty_attributes(
+            {
+                "name": name,
+                "editPermissions": edit_permissions,
+                "sharePermissions": share_permissions,
+                "description": description,
+            }
+        )
+        url = self._get_url("dashboard")
+        url = f"{url}/{id}/copy"
+        r = self._session.post(url, json=data)
+
+        raw_dashboard_json: dict[str, Any] = json_loads(r)
+        return Dashboard(self._options, self._session, raw=raw_dashboard_json)
+
+    @cloud
+    @experimental
+    def update_dashboard_automatic_refresh_seconds(
+        self, id: str, minutes: int
```

Comment:
update the name of the method or parameter to allign ... try to use the wording of the jira webui

Reply:
Thanks @studioj! I'll take a look again at the UI for this option and respond with a bit more context.

Reply:
Oh yeah, I see sorry about that! I'll change it to `refresh_minutes`:

<img width="612" alt="image" src="https://github.com/pycontribs/jira/assets/38542786/6df141e4-1a44-4414-accc-ae2c1dd8f670">

☝️ that's what it looks like in the UI. I'll push up the change momentarily.

Reply:
Fixed [here](https://github.com/pycontribs/jira/pull/1837/files#diff-442d14b20d591afd0c3de830d2803e30d4816eedc708f9ad129316aa7e5a4a7cR1296).